filters.py

```diff
@@ -11,3 +11,9 @@ def extendlinks(value):
     if value:
         value = URL_PATTERN.sub(r"<a href='\g<0>' class='truncated'>\g<0></a>", value)
     return value
+
+
+def titlefilter(title, reject_list=["Mr.", "Ms.", "Mrs."]):
```

Comment:
Use an immutable default. /ducks-like-a-linter
```suggestion
def titlefilter(title, reject_list=("Mr.", "Ms.", "Mrs.")):
```